common/logging_extra.py

```diff
@@ -62,8 +63,43 @@ def format_dict(self, record):
     return record_dict
 
   def format(self, record):
+    if self.swaglogger is None:
+      raise Exception("must set swaglogger before calling format()")
     return json_robust_dumps(self.format_dict(record))
 
+class SwagLogFileFormatter(SwagFormatter):
+  def fix_kv(self, k, v):
```

Comment:
docstring? What is the purpose of this function?

Reply:
overlap cannot exist in properties once expanded (e.g. `log.info` creates `msg` vs `log.event()` creates `msg.deviceState.deviceState.ambientTempC`)

Reply:
added comments explaining